compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -357,12 +353,16 @@ object Completion {
        sym
      }
 
-     /** Lists the symbols in this scope along with the name associated with them. */
-     def toListWithNames(implicit ctx: Context): List[(Symbol, Name)] = {
-       for {
-         (name, syms) <- nameToSymbols.toList
-         sym <- syms
-       } yield (sym, name)
+     /** Get the names that are known in this scope, along with the list of symbols they refer to. */
+     def mappings(implicit ctx: Context): Map[SimpleName, List[Symbol]] = {
```

Comment:
Couldn't the body of this method just be equal to `nameToSymbols` ? (assuming `def enter` above is changed to do `stripModuleClassSufffix`)

Reply:
Yes, good catch. That should have been the case from the beginning, since `nameToSymbold` contained exactly one symbol per name. Thanks